gemfile.d/test.rb

```diff
@@ -0,0 +1,7 @@
+group :test do
+  gem "minitest-tags"
```

Comment:
This hasn't had an update since 2012. Granted, it's tiny but I wonder if anyone actually uses this. It's only for developers to run specific tests.

Reply:
Proposing to drop it: https://github.com/Katello/katello/pull/10836